tasklist/client/src/common/utils/taskErrorHandling.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {z} from 'zod';
+
+const ErrorDataSchema = z.object({
+  title: z.string().optional(),
+  message: z.string().optional(),
+});
+
+export type ErrorData = z.infer<typeof ErrorDataSchema>;
+
+export const isTaskTimeoutError = (errorData: unknown): boolean => {
```

Comment:
Use a single export for the values and a single export for the types at the to follow the codebase style